profiling-ffi/src/lib.rs

```diff
@@ -9,6 +9,7 @@ use std::time::SystemTime;
 
 use chrono::{DateTime, TimeZone, Utc};
 
+mod array_queue;
```

Comment:
Would this make sense in `ddcommon-ffi` since there is nothing particularly profiling about it?